### PR TITLE
Fix: Footer in mobile

### PR DESCRIPTION
**Pull request by adodge-bcg**
# Description

This PR includes the following proposed change(s):

- Added bootstrap and jquery dependencies to fix footer in mobile

Consider importing bootstrap higher up if required in more components.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Locally and in storybook for shared, frontend, and demo

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: src/frontend/shared-components/src/footer/Footer.js [hunk 1]
```diff
@@ -27,7 +28,15 @@ export const FooterToggler = () => (
 export const Footer = () => (
   <footer className="footer">
     <nav className="navbar navbar-expand-sm navbar-dark justify-content-end">
-      <FooterToggler />
+      {/* <FooterToggler /> */}
```
Comment (akroon3r) [inline src/frontend/shared-components/src/footer/Footer.js:31]: is this commented out? if so, is it required?

Comment (adodge-bcg) [reply src/frontend/shared-components/src/footer/Footer.js]: I had done that for testing, should be fixed now

Comment (akroon3r) [reply src/frontend/shared-components/src/footer/Footer.js]: toggle on, my wayward son.

**Comment by akroon3r** [comment]
@sdevalapurkar-bcgov did you want to peep first before merge?

**Pull request merged**